sections/model.tex

```diff
@@ -0,0 +1,139 @@
+
+To follow in the suit of recent literature, a single node using a \(M|M|c\)
+queue is employed to model a hypothetical ward of patients presenting COPD.\ In
+addition to this, the grouping found in Section~\ref{subsec:overview} provides
+a set of patient classes in the queue. Under this model, the following
+assumptions are made:
+\begin{enumerate}
+    \item Inter-arrival and service times of patients are each exponentially
+        distributed with some mean. This is in spite of the system time
+        distributions shown in Figure~\ref{fig:clusters_los} in order to
+        simplify the model parameterisation.
+    \item There are $c$ servers available to arriving patients at the node
+        representing the overall resource availability including bed capacity
+        and medical staff.
+    \item There is no queue or system capacity. In~\cite{Williams2015}, a
+        queue capacity of zero is set under the assumption that any surplus
+        arrivals would be sent to another suitable ward or unit. As this
+        hypothetical ward represents COPD patients potentially throughout a
+        hospital, this assumption is not held.
+    \item Without the availability of expert clinical knowledge, a first-in
+        first-out service policy is employed in lieu of some patient priority 
+        framework.
+\end{enumerate}
+
+Each group of patients has its own arrival distribution. The parameter of this
+distribution is taken to be the reciprocal of the mean inter-arrival times for
+that group.
+
+Like arrivals, each group of patients has its own service time distribution.
+Without full details of the process order or idle periods during a spell, some
+assumption must be made about the true `service' time of a patient in hospital.
+It is assumed here that the mean service time of a group of patients may be
+approximated via their mean length of stay, i.e.\ the mean time spent in the
+system. For simplicity, this work considers the mean service time,
+\(\frac{1}{\mu}\), to be directly proportional to the mean total system time,
+\(\frac{1}{\phi}\), such that:
+\begin{equation}
+    \mu = p \phi
+\end{equation}
+
+\noindent where \(p \in \interval[open left]{0}{1}\) is some parameter to be
+determined for each group, denoted by \(p_i\) for group \(i\).
+
+In order to evaluate appropriate values of each \(p_i\) and the value of \(c\),
+the system is simulated across a parameter space. The objective then is to
+determine which set of parameters provides the most realistic analogue of the
+observed data. Given that the length of stay is one of the few ground truths
+available in the provided data, and given that length of stay and resource
+availability are connected, the output of this model is the simulated length of
+stay. With this simulated data, a parameter set can be evaluated by comparing
+its associated system time distribution with the real length of stay
+distribution.
```

Comment:
I can't exactly pin point what I don't like about this paragraph. Would something like the following work:

```suggestion
One of the few ground truths available in the provided data is the distribution 
of the total length of stay. Given that the length of stay and resource availability 
are connected, the approach here will be to simulate the length of stay distribution 
for a range of values \(p_i\) and \(c\) in order to find the parameters that best 
match the observed data.
```